### DISCUSSION TITLE
"Re-attach" to orphaned process?

### DISCUSSION THREAD

**Discussion by ferplnat** [microsoft/terminal#17348] [Q&A]
I use Terminal Canary and very occasionally (but often enough) things crash in a way that leave whatever I was using still running. Usually it's a text editor and I'd like to either pick up wherever it is or be able to get to it so I can gracefully close things and restart my session.

Is there some sort of way to tell Windows Terminal to try to attach to a process like pwsh.exe or even an orphaned OpenConsole.exe?
I tried to search for issues, but I may be framing my question incorrectly.

Being able to _intentionally_ detach and later reattach to stuff would be kind of cool, too.

**Comment by o-sdn-o** [comment]
You might want to try using some kind of terminal multiplexer like tmux or something like that. This approach may eliminate or at least redirect crashes to another plane, or maybe add more problems; this needs to be tested in a specific case.
